### DISCUSSION TITLE
Tax Included / excluded ItemPrice

### DISCUSSION THREAD

**Discussion by alx314** [amzn/selling-partner-api-models#3855] [General]
Hello 👋

I am trying to solve an issue of tax included / tax excluded prices with a third party developer. (Extensiv / Cartrover)
They built their solutions for all marketplaces to retrieve Sales orders and for my case link them to my accounting software.

They have built a logic where
• ItemTax+ItemPrice= Grand Total.
• If the order is tax inclusive, ItemTax should be at 0.
And I guess it works for various marketplaces and various clients of theirs.

But for my Japanese Marketplace account, all my payload have a Tax Inclusive ItemPrice and an amount for ItemTax.
![Screenshot 2024-04-25 at 10 16 01](https://github.com/amzn/selling-partner-api-docs/assets/82567739/b88c0cac-edf8-46d3-b40b-6f484bc0f70a) 
(The product is 2700JPY tax excluded + 10% tax so 270JPY = 2970JPY tax included)

Questions:
• Is this a behaviour that is not shared across marketplaces? across accounts?
• Is there an API request that gives the info “ItemPrice are tax inclusive/exclusive” or “this account processes taxes this or that way”

Thank you for your help!
Alex.

**Comment by DavisuaCoder** [comment]
Hey i have the same problem, did you find any answer??
